docs/deployment/visual-studio-properties.md

```diff
@@ -0,0 +1,666 @@
+
+- **Windows Targets**
+
+  If you're going to target Windows, these settings describe the target Windows version.
+
+  | Setting | Default value | Description |
+  | - | - | - |
+  | Target the Windows platform | Checked | When checked, the .NET MAUI project will target and build a Windows version of your app. Uncheck to disable the Windows target. |
+  | Target Windows Framework | `net6.0-windows10.0.19041.0` | The [Target Framework Moniker][tfm] used to target Windows.
+  | Minimum Target Windows Framework | `10.0.17763.0` | The minimum version of Windows your app targets.
+
+## Build
+
+The **Build** section describes settings related to compiling your app.
+
+### General
+
+Settings related to target platforms.
+
+- **Conditional compilation symbols**
+
+  Specifies symbols on which to perform conditional compilation. Separate symbols with a semicolon `;`. Symbols can be broken up into target platforms
+
+- **Platform target**
+
+  Specifies the processor to be targeted by the output file. Choose `Any CPU` to specify that any processor is acceptable, allowing the application to run on the broadest range of hardware.
+
+  Typically this is set to `Any CPU` and the runtime identifier setting is used to target a CPU platform.
+
+  | Option | Description |
+  | ------ | ----------- |
+  | `Any CPU` | (Default) Compiles your assembly to run on any platform. Your application runs as a 64-bit process whenever possible and falls back to 32-bit when only that mode is available. |
+  | `x86` | Compiles your assembly to be run by the 32-bit, x86-compatible runtime. |
+  | `x64` | Compiles your assembly to be run by the 64-bit runtime on a computer that supports the AMD64 or EM64T instruction set. |
+  | `ARM32` | Compiles your assembly to run on a computer that has an Advanced RISC Machine (ARM) processor. |
+  | `ARM64` | Compiles your assembly to run by the 64-bit runtime on a computer that has an Advanced RISC Machine (ARM) processor that supports the A64 instruction set. |
+
+- **Nullable**
+
+  Specifies the project-wide C# nullable context. For more information, see [Nullable References](/dotnet/csharp/nullable-references#nullable-contexts).
+
+  | Option        | Description |
+  | ------------- | ----------- |
+  | Unset         | (Default) If this setting isn't set, the default is `Disable`. |
+  | `Disable`     | Nullable warnings are disabled. All reference type variables are nullable reference types. |
+  | `Enable`      | The compiler enables all null reference analysis and all language features. |
+  | `Warnings`    | The compiler performs all null analysis and emits warnings when code might dereference null. |
+  | `Annotations` | The compiler doesn't perform null analysis or emit warnings when code might dereference null. |
+
+- **Implicit global usings**
+
+  Enables implicit global usings to be declared by the project SDK. This is enabled by default. Imports many of the .NET MAUI namespaces automatically to all code files.
```

Comment:
Wondering if this should be reworded to something to clarify users won't see `using` statements added to their code files?